packages/wonder-blocks-data/src/util/types.js

```diff
@@ -34,9 +44,29 @@ export type InterceptContextData = {
     ...
 };
 
+/**
+ * A cache of fulfilled request responses.
+ */
 export type Cache = {
     [key: string]: CacheEntry<any>,
     ...
 };
```

Comment:
Ideally, this would get renamed to ResponseCache or something, but then we'd have to rename other things that are already called that. Naming things is hard. For now, I decided to leave the existing types as they are for this change.

Reply:
See https://github.com/Khan/wonder-blocks/pull/1341?w=1 where I fix up the naming. :)